.github/workflows/ci_tests.yml

```diff
@@ -97,7 +101,10 @@ jobs:
             cmake_args:
               description: "TSan"
 
-    name: "Unit: ${{ matrix.platform.description }} ${{ matrix.cpp_version }} ${{ matrix.cmake_args.description }}"
+    name: "Unit:
+              ${{ matrix.platform.description }}
+              ${{ matrix.cpp_version }}
+              ${{ matrix.cmake_args.description }}"
```

Comment:
I edited this line and a few others like it to make the file easier to read, diff, and review. Hopefully everyone's OK with doing that as part of this PR.